invenio_app_ils/records/jsonresolver/internal_location.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Ils Internal Location record resolver."""
+
+import jsonresolver
+from flask import current_app
+from invenio_pidstore.errors import PersistentIdentifierError
+
+from ..api import InternalLocation
+
+
+@jsonresolver.route(
+    "/api/internal-locations/<pid_value>", host="ils.mydomain.org"
```

Comment:
I did the part the harcoding the host, but it is dangerous: if we override the config and change the host/scheme, this will brake.

Same for all the others

Reply:
I also think we should change this in order not to conflict with invenio-records-rest.